### PR TITLE
Added Custom Connector NSF Data

### PR DESCRIPTION
**Pull request by DataboatDev**
NSF Data Connector provides access to any IBM / HCL Domino database for which Domino Access Services (DAS) are enabled. The Connector represents databases, views, view entries, and documents in JSON format.
---
- [+] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [+] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@DataboatDev Can you please move the connector from `custom-connectors` to `certified-connectors`

**Comment by DataboatDev** [comment]
> Can you please move the connector from `custom-connectors` to `certified-connectors`
@sriyen-msft My understanding is that the `certified-connectors` is for certified connectors. Ours is not yet. We are in the certification process and it gets delayed at the moment. So we decided to go ahead with the custom connector and get back to certification once all parties are ready.

File: custom-connectors/NSF Data/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Dynamic host",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('host')"
+        }
+      }
+    ],
+    "publisher": "Databoat AG"
```
Comment (sriyen-msft) [inline custom-connectors/NSF Data/apiProperties.json:55]: @DataboatDev Please add `stackOwner` property as well.

Documentation - [Add Metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: custom-connectors/NSF Data/apiProperties.json (hunk 1 above)
Comment (DataboatDev) [reply custom-connectors/NSF Data/apiProperties.json]: @sriyen-msft Done.

File: custom-connectors/NSF Data/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      {
+        "templateId": "dynamichosturl",
+        "title": "Dynamic host",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('host')"
+        }
+      }
+    ],
+    "publisher": "Databoat AG",
+    "stackOwner": "HCL Technologies"
```
Comment (sriyen-msft) [inline custom-connectors/NSF Data/apiProperties.json:56]: @DataboatDev Can you please let us know the relationship b/w `Databoat AG` and `HCL Technologies`. If the connector is being published by `Databoat AG`, the `stackOwner` should be the same.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: custom-connectors/NSF Data/apiProperties.json (hunk 2 above)
Comment (DataboatDev) [reply custom-connectors/NSF Data/apiProperties.json:56]: @sriyen-msft This is a generic Connector that allows communication with HCL Domino on-premise installations. Due to NDA I cannot provide more details here. Can we take it offline? Perhaps you can find more details in your CRM. Communications were done with a colleague of you who unfortunately just left your team.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**